cmds/init/init.go

```diff
@@ -31,6 +31,10 @@ var (
 )
 
 func main() {
+	defer func() {
+		log.Printf("well, this sucks, init exited. hanging forever")
+		for {}
```

Comment:
If I understand correctly, this would have the side effect of heating your CPUs and discharging your batteries faster.

Also, if you are using a chroot or a container, it is not as easy to stop (you have to use kill).